docs/style_guide/definitions.md

```diff
@@ -0,0 +1,103 @@
+# Definitions (Top level definitions)
+
+This section will focus on styling the top level definitions available in the Ballerina.
+
+## Indentation
+* No indentation applied for the top level constructs.
+
+```ballerina
+import ballerina/http;
+
+function func1() {
+    int a = 0;
+}
+```
+
+## Imports
+
+* `import` keyword followed by a single space.
+* No spaces between org name , divider(`/`) and package name.
```

Comment:
package name should be changed to module name

Reply:
will change